contracts/TopShot.cdc

```diff
@@ -679,9 +680,15 @@ pub contract TopShot: NonFungibleToken {
                         UInt64(self.data.serialNumber)
                     )
                 case Type<MetadataViews.Royalties>():
+                    let royaltyReceiver: Capability<&{FungibleToken.Receiver}> =
+                        getAccount(0xTOPSHOTROYALTYADDRESS).getCapability<&AnyResource{FungibleToken.Receiver}>(MetadataViews.getRoyaltyReceiverPublicPath())
```

Comment:
Presumably we'll need to run a tx to add a receiver at this address?

Reply:
I think we should leave it as-is, because it still does allow reference to the address, and will lead to the proper path eventually. There currently aren't any marketplaces using this view yet, and either marketplaces should use the address to find the proper vault or hopefully we'll get the switchboard ready before then (Which is more likely the case.)

One thing we should do - on these topshot royalty receiving accounts, we will need to setup a vault for FUSD, USDC, and Switchboard when available.